golos.publication/golos.publication.abi

```diff
@@ -310,7 +310,7 @@
     "base": "",
     "fields": [
       {
-        "name": "post_id",
+        "name": "postid",
```

Comment:
There is no need to rename fields/variables.
Only underscore in `N()` macro is an issue.
`post_id` in field or variable name is good for readability.